wagtail/wagtailsearch/signal_handlers.py

```diff
@@ -1,10 +1,14 @@
 from __future__ import absolute_import, unicode_literals
 
+import logging
+
 from django.db.models.signals import post_delete, post_save
 
-from wagtail.wagtailsearch.backends import get_search_backends
+from wagtail.wagtailsearch.backends import get_search_backends_with_name
 from wagtail.wagtailsearch.index import get_indexed_models
 
+logger = logging.getLogger('wagtail.search.auto_reindex')
```

Comment:
The usual done thing is to do `logger = logging.getLogger(__name__)`. If Wagtail is using custom named loggers, is there any documentation on which custom loggers exist, and what they do? Actually this information would be good regardless of whether the loggers have a custom name, but is probably out of scope for this PR.